src/lumigo_tracer/libs/custom_json_dump.py

```diff
@@ -0,0 +1,80 @@
+from json.encoder import (
```

Comment:
Note this is a lib file that was copied from StackOverflow (with small changes).
We're using it in order to not have any external libraries.